appinventor/components/src/com/google/appinventor/components/common/ComponentDescriptorConstants.java

```diff
@@ -0,0 +1,27 @@
+package com.google.appinventor.components.common;
```

Comment:
@ZhengCornell Make sure to include the preamble comments here as seen in other files.